gpu/assets.go

```diff
@@ -17,23 +17,29 @@ import (
 )
 
 var (
-	lock        sync.Mutex
-	payloadsDir = ""
+	lock       sync.Mutex
+	runnersDir = ""
```

Comment:
Is the global state needed?

Reply:
Prior to the packaging changes on linux, we had a ~cycle in the dependencies where the GPU discovery logic wants to leverage the library from the payload to attempt discovery.  Now that this is always carried in the tar file and not a Go binary payload, I should be able to do a refactoring pass and just always assume the location is relative to the executable (or use the system versions).  My inclination would be to do this in a follow up PR after we get 0.3.7 out.

Reply:
When we are extracting payloads into a temp dir, it has a random string, so we need to keep track of that through the life of the process to avoid expanding the payloads over and over.